docs/user/alerting/alert-types.asciidoc

```diff
@@ -0,0 +1,115 @@
+[role="xpack"]
+[[alert-types]]
+== Alert types
+
+{kib} supplies alerts types in two ways: some are built into {kib}, while domain-specific alert types are registered by {kib} apps such as <<xpack-apm,*APM*>>, <<xpack-infra,*Metrics*>>, and <<xpack-uptime,*Uptime*>>.
+
+This section covers built-in alert types. For domain-specific alert types, refer to the documentation for that app. 
+
+Currently {kib} provides one built-in alert type: the <<alert-type-index-threshold>> type. 
+
+[float]
+[[alert-type-index-threshold]]
+=== Index threshold
+
+The index threshold alert type is designed to run an {es} query over indices, aggregating field values from documents, comparing them to threshold values, and scheduling actions to run when the thresholds are met.
+
+[float]
+==== Creating the alert
+
+An index threshold alert can be created from the *Create* button in the <<alert-management, alert management UI>>. Fill in the <<defining-alerts-general-details, general alert details>>, then select *Index Threshold*.
```

Comment:
How about:

To create an index threshold alert, click **Create**, fill in the genera alert details, then click *Index Threshold*.